test/GithubApi.Users.test.js

```diff
@@ -0,0 +1,69 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const responseTime = require('superagent-response-time');
+const { expect } = require('chai');
+
+const urlBase = 'https://api.github.com';
+
+describe.only('For getting all users', () => {
+  let queryTime;
+  let usersPagination;
+
+  before(() => {
+    agent
+      .get(`${urlBase}/users`)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .then((response) => {
+        usersPagination = response.body.length;
+      });
```

Comment:
please add a .then with next promise as a return param, then delete the brackets since they are not necessary